integrations/FacebookPixel/browser.js

```diff
@@ -199,7 +200,7 @@ class FacebookPixel {
           customProperties
         ),
         {
-          eventID: messageId,
+          event_id: getEventId(rudderElement.message),
```

Comment:
Get the event ID once in the beginning of the function and re-use it everywhere.

Reply:
Done